### PR TITLE
@damassi => [ArtworkBrick] Link tombstone info to the artwork page for the new brick

### PR DESCRIPTION
**Pull request by mzikherman**
We link each line in the tombstone info out to the relevant areas: artist page for the artist line, partner profile page for the partner line, etc.

This suppresses all those links, and instead wraps the entire tombstone info in a link to the artwork page, if you have the lab feature.

### CODE REVIEW

File: src/Components/Artwork/Details.tsx [hunk 1]
```diff
@@ -38,9 +41,9 @@ export class Details extends React.Component<Props, null> {
         .reduce((acc, artist, index) => {
           return acc.concat([
             ", ",
-            <TextLink href={artist.href} key={artist.__id + "-" + index}>
-              {artist.name}
-            </TextLink>,
+            includeLinks
```
Comment (damassi) [inline src/Components/Artwork/Details.tsx]: Maybe add a `FIXME: Remove BNMO lab feature`?

File: src/Components/Artwork/Metadata.tsx [hunk 2]
```diff
@@ -2,6 +2,7 @@ import { Metadata_artwork } from "__generated__/Metadata_artwork.graphql"
 import { ContextConsumer } from "Artsy/Router"
 import colors from "Assets/Colors"
 import { garamond } from "Assets/Fonts"
+import StyledTextLink from "Components/TextLink"
```
Comment (damassi) [inline src/Components/Artwork/Metadata.tsx]: What is this component?

**Comment by artsyit** [comment]
Deploy preview for *artsy-reaction* ready!

Built with commit 14f1bd4a1cc8704cd4c605a84e5402a240cc7a49

https://deploy-preview-1298--artsy-reaction.netlify.com

File: src/Components/Artwork/Metadata.tsx [hunk 3]
```diff
@@ -32,13 +33,25 @@ export class MetadataContainer extends React.Component<MetadataProps> {
...
+                artwork={artwork}
+              />
               {!enableLabFeature &&
                 extended && <ContactBlock artwork={artwork} />}
             </div>
           )
+          return enableLabFeature ? (
+            <StyledTextLink href={artwork.href}>
+              {detailsContent}
+            </StyledTextLink>
```
Comment (damassi) [inline src/Components/Artwork/Metadata.tsx]: What happens if this is just `<Serif><a href='#'>...</a></Serif>`? I (think) StyledTextLink refers to our old look, before we implemented the new design system

File: src/Components/Artwork/Metadata.tsx (hunk 2 above)
Comment (mzikherman) [reply src/Components/Artwork/Metadata.tsx]: This makes the tombstone info be a plain link, but without all this other 'cruft', like underlining on hover.

Making it `<a>` seemed to add a bunch of that stuff, and I couldn't figure out how to remove that. So this actually works great!

File: src/Components/Artwork/Metadata.tsx (hunk 3 above)
Comment (mzikherman) [reply src/Components/Artwork/Metadata.tsx]: answered above, but I was using `StyledTextLink` b/c that seemed to do what I want, which is: no underlining or underlining on hover, and you get the nice affect of the text going darker when you do hover, which matches the existing behavior as well.

With just a plain `<a>` tag I _was_ getting all that extra cruft (and no color change on hover).

Comment (damassi) [reply src/Components/Artwork/Metadata.tsx]: Cool cool, thanks for explaining 👍

**Comment by damassi** [comment]
👍

**Pull request merged**